### PR TITLE
First draft of net-a-porter case study

### PR DESCRIPTION
**Pull request by PaulKinlan**

### CODE REVIEW

**Comment by jpmedley** [comment]
@PaulKinlan Looking at this now.

File: src/content/en/showcase/case-study/net-a-porter.markdown [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+<blockquote>"Building using web 
+standards keeps our code close to the platform. This makes it easier to 
+learn how to use and build new components, ensuring performance will 
+continually get better as browsers improve, and letting us plan for the
+long-term. Because it’s based on the W3C standard, we are confident in 
+building on the platform. Soon even the need for polyfills will broadly go away."</blockquote>
+
+And there were SEO benefits, too. Glen says that the 
+
+<blockquote>"components
```
Comment (jpmedley) [inline src/content/en/showcase/case-study/net-a-porter.markdown]: Capital c.

File: src/content/en/showcase/case-study/net-a-porter.markdown [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+effectively surface detailed structured data about products to search 
+engines. Testing this structured data is now directly part of our team's 
+continuous integration scheme."</blockquote>
+
+By making the up-front investment in designing and building components,
+the NET-A-PORTER team is seeing dramatic long-term efficiency gains: 
+New pages and features are simpler and faster to create and launch, and
```
Comment (jpmedley) [inline src/content/en/showcase/case-study/net-a-porter.markdown]: Lower case n on 'New pages'.

File: src/content/en/showcase/case-study/net-a-porter.markdown [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+</blockquote>
+
+Glen feels there is much more to explore with Polymer. He says, 
+
+<blockquote>"This
+is just the beginning of our Polymer journey. We are currently improving 
+the delivery and maintainability of existing components, while working 
+towards expanding the use of Polymer to more NET-A-PORTER pages. We 
+are working towards documenting our approach with a publicly available 
+NET-A-PORTER component style guide. Polymer has also been very useful
```
Comment (jpmedley) [inline src/content/en/showcase/case-study/net-a-porter.markdown]: We need to remember to come back and link to that when it comes online.

File: src/content/en/showcase/case-study/net-a-porter.markdown [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+is just the beginning of our Polymer journey. We are currently improving 
+the delivery and maintainability of existing components, while working 
+towards expanding the use of Polymer to more NET-A-PORTER pages. We 
+are working towards documenting our approach with a publicly available 
+NET-A-PORTER component style guide. Polymer has also been very useful 
+for internal tooling. We recently built a set of graphing components and 
+re-platformed our entire monitoring service to use Polymer."
+</blockquote>
+
+## Results
```
Comment (jpmedley) [inline src/content/en/showcase/case-study/net-a-porter.markdown]: This section: (1) add periods to the bullets. (2) Maybe make these a tl:dr; at the top.

**Comment by jpmedley** [comment]
@PaulKinlan I think that's the cleanest text I've ever seen from you. Good job. I did find one or two things.

**Comment by PaulKinlan** [comment]
I didn't write it :)

**Pull request merged**